### ISSUE TITLE
Better feedback when uploading incorrect data

### ISSUE THREAD

**Issue by wouterbeek** [pldn/LDWizard#29]
# Observation

It is currently possible to upload incorrect data.  E.g., a CSV table where some data rows contain more/less cells than the header cell.

# Suggestion

Let's emit an error whenever the input data is not well-formed.  Preferably, communicate the reason why the data is incorrect, and/or display the snippet from the source data where the issue was first detected (allowing the user to gain insight as to why their source data is incorrect.

**Comment by philipperenzen** [comment]
Resolved in version 2.3.3

**Comment by EnnoMeijers** [comment]
The newly added validation of the uploaded csv is too strict. It doesn't allow empty values in a cell any more. This is too strict in my opinion as there can be valid reasons for not have data in a particular cell, it could be optional data. And it can certainly be the case when working with refinements and reprocessing the enhanced version of the csv. The will always contain incomplete data because refinements will never be available for every value in the input data. The original problem was about inconsistencies in header and row data.

**Comment by philipperenzen** [comment]
This was resolved in version 3.0.2, only CSV column inaccuracy will be checked and empty row cells will not result in an error when present in the CSV data. 

### Wrong CSV data:
```csv
col_1,col_2,col_3
error_1,error_2
error_1,error_2,error_3,error_3
```

### Correct CSV data:
```csv
col_1,col_2,col_3
correct_1,"",""
"","",""
```
